apps/graduate/src/widgets/sidebar/ui/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from '@tanstack/react-router';
+import { adminMenuSections } from '../model/model';
+import * as s from './Sidebar.css';
+
+export const Sidebar = () => {
```

Comment:
저희 컨벤션 정한대로 컴포넌트 만들어주세요.
```suggestion
export default function Sidebar() {
```